TanukiKitTests/ProjectTests.swift

```diff
@@ -35,11 +35,11 @@ class ProjectTests: XCTestCase {
 
     func testFailToGetProjects() {
         let json = "{\"message\":\"401 Unauthorized\"}"
-        let session = TanukiKitURLTestSession(expectedURL: "https://gitlab.com/api/v3/projects?page=1&per_page=100", expectedHTTPMethod: "GET", response: json, statusCode: 401)
```

Comment:
I tested all the requests manually, they all work with empty parameters, the problem is we support so many parameters, if I were to do an if then else for each possible combination of used parameters (like in OctoKit) it would take me exactly 3,628,800 if statements.